.github/workflows/release.yaml

```diff
@@ -69,3 +69,30 @@ jobs:
           asset_name: micromissiles-${{ github.event.workflow_run.head_branch }}-linux-x86_64.tar.gz
           asset_path: build/build-StandaloneLinux64.tar.gz
           asset_content_type: application/gzip
+      - name: Upload Windows plugins
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_name: micromissiles-plugins-${{ github.event.workflow_run.head_branch }}-windows-x86_64.tar.gz
+          asset_path: build/plugins-windows-latest.tar.gz
```

Comment:
These will all fail because the artifact is named `plugins.tar.gz` for all platforms.  I think the easiest solution is to rename `plugins.tar.gz` prior to uploading them when building the plugins.  In `build.yaml`, this means we can download the artifacts with `merge-multiple: false` into `Assets/Plugins`.

Reply:
Fixed in https://github.com/PisterLab/micromissiles-unity/pull/40/commits/2456fec2cfe76486b477e3390184cf13b31e2418.